tests/assessment_store_tests/conftest.py

```diff
@@ -38,6 +46,73 @@ def ctdf_application():
     yield copy.deepcopy(ctdf_input_data)
 
 
+def bulk_insert_application_record(
```

Comment:
The other methods in here seem to work as `@pytest.fixtures` where they accept the database connection used by the running tests, do some things and yield a result. I think ideally this would work in the same way but if we think this will only be around for a short while this should be fine?

Reply:
Yeah I didn't want to mess with that too much as I think we have a TODO around sorting out unit test data, and also this inserts into assessment which will hopefully soon be one and the same as apply